### ISSUE TITLE
Color picker partially hidden in Symbolizer Editor

### ISSUE THREAD

**Issue by rdewit** [geostyler/geostyler-demo#169]
# Bug

**Describe the bug**
When selecting the color picker in the Symbolizer Editor, it does not show fully.

**To Reproduce**
1. Go to https://geostyler.github.io/geostyler-demo/
2. Click on the Symbolizer for the first rule
3. Click on 'Pick' in 'Stroke-Color'
4. Observe that the color picker is partially hidden

**Expected behavior**
The color picker should be fully visible.

**Screenshots**
See here:
![color-picker-hidden](https://user-images.githubusercontent.com/219691/88886218-50689e80-d27d-11ea-8d38-e0a15eb5cdaa.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Chrome version 84, Firefox version 79

**Additional context**
Where should this problem be solved: in the demo or in geostyler itself? Is there a reason why the `position` is set to 'absolute' in [ColorField.less](https://github.com/geostyler/geostyler/blob/master/src/Component/Symbolizer/Field/ColorField/ColorField.less#L30)?

A quick workaround would be to add the following to App.css in geostyler-demo:

```css
.gs-symbolizer-editor .sketch-picker {
  position: relative;
}
```
But that may not be the right way to solve this. Any thoughts?

**Comment by rdewit** [comment]
Could it be a regression of this fix: geostyler/geostyler#333?

**Comment by marcjansen** [comment]
Thanks for the report, we'll surely take a look.

**Comment by rdewit** [comment]
The recent fix in geostyler/geostyler#1302 has fixed this issue. Thanks, @mholthausen! Closing.
